addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -1643,7 +1644,7 @@ export function isSelectionFormat(editable, format) {
     const selectedNodes = getTraversedNodes(editable)
         .filter(n => n.nodeType === Node.TEXT_NODE);
```

Comment:
I would add this I think i'ts better to get only the nodes we need from the beginning, I tested with it and all tests passed too
```suggestion
    const selectedNodes = getTraversedNodes(editable)
        .filter(n => n.nodeType === Node.TEXT_NODE && n.nodeValue !== INVISIBLE_BOM_CHAR);
```

Reply:
If we consider only `\ufeff` in a selection, then the `isSelectionFormat` will return false because of `selectedNodes.length` which will have 0. we should keep those nodes and then 
`selectedNodes.every(n => n.nodeValue === INVISIBLE_BOM_CHAR || isFormatted(n, editable))` to consider them as formatted.

Reply:
Understood, thanks for the explanation !